nuitka/build/static_src/CompiledCodeHelpers.c

```diff
@@ -541,6 +541,42 @@ PyObject *BUILTIN_LEN(PyObject *value) {
     return PyInt_FromSsize_t(res);
 }
 
+PyObject *BUILTIN_ANY(PyObject *value) {
+    CHECK_OBJECT(value);
+
+    PyObject *it = PyObject_GetIter(value);
+    PyObject *(*iternextfunc)(PyObject *) = *Py_TYPE(it)->tp_iternext;;
+    PyObject *item;
+
+    if (unlikely((it == NULL)))
+        return NULL;
+
+    int cmp;
+
+    for (;;) {
+        item = iternextfunc(it);
+
+        if (unlikely((item == NULL)))
+            break;
+        cmp = PyObject_IsTrue(item);
+        Py_DECREF(item);
+        if (unlikely(cmp < 0)) {
+            Py_DECREF(it);
+            return NULL;
+        }
+        if (cmp > 0) {
+            Py_DECREF(it);
+            Py_INCREF(Py_True);
+            return Py_True;
+        }
+    }
+    Py_DECREF(it);
+    if (unlikely(!CHECK_AND_CLEAR_STOP_ITERATION_OCCURRED()))
```

Comment:
An important style preference, please always use blocks with if statements. The `if () {}` is such a huge bug avoider, and readability aid, it is worth the extra characters.